recipes/evalhyd-python/meta.yaml

```diff
@@ -0,0 +1,76 @@
+# name for conda package (i.e. `conda install -c conda-forge <project_name>`)
+{% set project_name = "evalhyd-python" %}
+# name for python package (i.e. `python -c "import <package_name>"`)
+{% set package_name = "evalhyd" %}
+{% set version = "0.1.0.0" %}
+{% set sha256 = "339fe203c6dea5a8d5005de3f2dbe45019d9603508c238ba01ca0d80375fe452" %}
+
+package:
+  name: {{ project_name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.irstea.fr/HYCAR-Hydro/evalhyd/{{ project_name }}/-/archive/v{{ version }}/{{ project_name }}-v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script_env:
+    - CPPFLAGS=-D_LIBCPP_DISABLE_AVAILABILITY -std=c++17    # [osx]
+    - EVALHYD_PYTHON_VENDOR_XTL=FALSE
+    - EVALHYD_PYTHON_VENDOR_XTENSOR=FALSE
+    - EVALHYD_PYTHON_VENDOR_XTENSOR_PYTHON=FALSE
+    - EVALHYD_PYTHON_VENDOR_EVALHYD_CPP=FALSE
```

Comment:
Better to set these in the build scripts directly and avoid `script_env`

Reply:
This has the advantage of not having to duplicate the information in two separate build files?

Reply:
Yes